pkgs/games/openttd/grfcodec.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchFromGitHub, boost, cmake, git }:
+
+stdenv.mkDerivation rec {
+  pname = "openttd-grfcodec";
+  version = "6.0.6+git20210310";
```

Comment:
```suggestion
  version = "unstable-2021-03-10";
```

Reply:
Done, although I find the use of "unstable-{date}" rather than "{basever}-{date}" (which is used elsewhere in nixpkgs) objectionable, since it doesn't sort cleanly with the release versions.

Reply:
I agree with you. But that's how it is done in nixpkgs 🤷. And we don't need to sort the package versions, we are not Debian :)